R/utils-problems.R

```diff
@@ -3,10 +3,9 @@
 #' See [readr::problems()] for details.
 #'
 #' @name problems
-#' @rdname problems
-#' @keywords internal
-#' @export
 #' @importFrom readr problems
+#' @export
+#' @noRd
```

Comment:
@PietrH I was wrong. Not creating an `rd` will result in a CHECK warning. Fixed in 861d808. The "Help on topic 'problems' was found in the following packages:" is thus unavoidable.